db/people.yml

```diff
@@ -2039,3 +2039,219 @@ zzgong:
+      group: bg
+      advisor: sbillinge
+  employment:
+    - begin_year: 2011
+      position: Assistant Scientist
+      organization: Brookhaven National Laboratory
+    - begin_year: 2001
+      position: Research Associate
+      organization: msu
+      status: phd
+      end_year: 2003
+      end_month: 5
+      group: bg
+  name: Emil Bozin
+  position: Assistant Scientist
+xiangyun:
+  active: false
+  aka:
+    - Xiangyun
+    - Xiangyun Qiu
+  avatar: https://static-exp1.licdn.com/sc/h/djzv59yelk5urv2ujlazfyvrk
+  bio: bio
+  education:
+    - begin_year: 1999
+      degree: PhD
+      institution: msu
+      end_year: 2004
+      group: bg
+      advisor: sbillinge
+  employment:
+    - begin_year: 2017
+      position: Assistant Professor
+      organization: georgewashingtonu
+    - begin_year: 1999
+      position: Research Associate
+      organization: msu
+      status: phd
+      end_year: 2004
+      group: bg
+  name: Xianhyun
+  position: Assistant Professor
+mouathshatnawi:
+  active: false
+  aka:
+    - Mouath Ghazi Shatnawi
+    - Mouath Shatnawi
+  avatar: https://static-exp1.licdn.com/sc/h/djzv59yelk5urv2ujlazfyvrk
+  bio: bio
+  education:
+    - begin_year: 2001
+      degree: PhD
+      institution: msu
+      end_year: 2007
+      end_month: 5
+      group: bg
+      advisor: sbillinge
+  employment:
+    - begin_year: 2016
+      position: Assistant Professor
+      organization: Hashemite University
+    - begin_year: 2001
+      position: Research Associate
+      organization: msu
+      status: phd
+      end_year: 2007
+      end_month: 5
+      group: bg
+  name: Mouath Shatnawi
+  position: Assistant Professor
+moneedshatnawi:
+  active: false
+  aka:
+    - Moneeb Shatnawi
+  avatar: https://static-exp1.licdn.com/sc/h/djzv59yelk5urv2ujlazfyvrk
+  bio: bio
+  education:
+    - begin_year: 2004
+      degree: PhD
+      institution: msu
+      end_year: 2007
+      end_month: 7
+      group: bg
+      advisor: sbillinge
+  employment:
+    - begin_year: 2008
+      position: Assistant Professor
+      organization: University of Jordan
+    - begin_year: 2004
+      position: Research Associate
+      organization: msu
+      status: phd
+      end_year: 2007
+      end_month: 7
+      group: bg
+  name: Moneed Shatnawi
+  position: Assistant Professor
+hyavas:
+  active: false
+  aka:
+    - Hasan Yavas
+  avatar: https://static-exp1.licdn.com/sc/h/djzv59yelk5urv2ujlazfyvrk
+  bio: bio
+  education:
+    - begin_year: 2001
+      degree: PhD
+      institution: msu
+      end_year: 2007
+      end_month: 10
+      group: bg
+      advisor: sbillinge
+  employment:
+    - begin_year: 2018
+      position: Staff Scientist
+      organization: SLAC National Accelerator Laboratory
+    - begin_year: 2001
+      position: Research Associate
+      organization: msu
+      status: phd
+      end_year: 2007
+      end_month: 10
+      group: bg
+  name: Hasan Yavas
+  position: Staff Scientist
+tdykhne:
```

Comment:
timur changed his last name to Davies, so I think he wants his name as Timur Davies but put Timur Dykhne in the aka